bin/BuildSystem/BuildSystemBase.py

```diff
@@ -240,18 +240,17 @@ def internalPostInstall(self):
 
             regexp = re.compile('{.*} (.*)')
             for f in files:
```

Comment:
`if CraftCore.cache.findApplication("peparser"):`

Reply:
I have that check a few lines above?
-> https://github.com/KDE/craft/blob/b5d8444910e59e4ba0987353b581bbcdfa95a153/bin/BuildSystem/BuildSystemBase.py#L237

Reply:
I see